### ISSUE TITLE
Webpage field saving incorrectly

### ISSUE THREAD

**Issue by HeidiEllis** [pods-framework/pods#3993]
I have a simple POD to represent learning materials that contains a link to a web page:

![image](https://cloud.githubusercontent.com/assets/533364/22910974/7e13e7d0-f22b-11e6-9237-7ed86d920948.png)

However, when I Update the POD with a link to another POD as shown in the image above, it somehow corrupts the "Overview" text box above it. Note the "http://" inserted before the text for the Overview.  There is also a forward slash appended to the end of the Overview text. 

![image](https://cloud.githubusercontent.com/assets/533364/22911108/0f2105e6-f22c-11e6-8944-7a5080172bcb.png)

In addition, the link does not display when I display the POD: 
![image](https://cloud.githubusercontent.com/assets/533364/22911166/58d56b3c-f22c-11e6-883a-05a5676cbbaf.png)

PODs with links not to other PODs work fine. I've tried this in both Chrome Version 51.0.2704.63 (64-bit) and Firefox 46.0.1 and get the same behavior.

**Comment by HeidiEllis** [comment]
Whoops, this was my bad. The type of the "Overview" was webpage and it was inserting the http://  I changed the type and this eliminated the "http://", but the link still does not display.

**Comment by jimtrue** [comment]
@HeidiEllis For clarification, is the field acting the way you intended or is something still not functioning as you expect the field to function? (I see your note 'the link still does not display'). Typically when you're doing a link in a template, you'd need to do the wrapping `<a href="{@link}">Link Text</a>` yourself.

**Comment by HeidiEllis** [comment]
@jimtrue Thank you for the swift response! There were two different things going on. One was my mistake which was that I had the incorrect slug for the link name in the learning material template.  When I fixed that, the link appeared correctly. 

The second is what I think is a bug, which unfortunately I can't seem to reproduce. When I included an incorrect link to a pod located on the same site, for some reason the system was inserting the "http://" before the overview which was the field before the link field. It works correctly now that I have fixed the slug name in the template.

I tried to reproduce the error by going back and changing the slug name to the wrong name, but that didn't seem to reproduce the error. It just (correctly) didn't display the link. So I'm guessing that the odd behavior is somehow related to having the incorrect slug name in the template, but I can't reproduce the problem. That's why I closed the issue.
